src/Customizer.php

```diff
@@ -120,6 +125,18 @@ public function register() {
 			'section'  => 'sticky_menu_section',
 		) );
 
+		$this->wp_customize->add_control( 'sticky_menu_layout', array(
```

Comment:
@krumpak the control name is still "layout", this should be changed to "visibility"